common/interfaces/desktop_app_interface/usb_event.c

```diff
@@ -197,17 +258,36 @@ bool usb_get_event(usb_event_t *evt) {
     return false;
   }
 
+  size_t request_type = 0;
   reset_event_obj(evt);
 
   if (usb_event.flag) {
-    core_error_type_t status = validate_msg_context(core_msg);
+    core_error_type_t status = get_core_req_type(core_msg, &request_type);
     if (CORE_NO_ERROR != status) {
       // now clear event as it is not supposed to reach the app
       usb_clear_event();
       // TODO: send an error to host
```

Comment:
can we resolve this todo now? maybe we could use `send_core_error_msg_to_host` to send error to host.

Reply:
updated